src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -67,10 +68,20 @@ public ReadOnlyPerson getPerson() {
     public CommandResult execute() {
         try {
             addressBook.addPerson(toAdd);
-            return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
+            return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd),listAfterAdd());
         } catch (UniquePersonList.DuplicatePersonException dpe) {
             return new CommandResult(MESSAGE_DUPLICATE_PERSON);
         }
     }
 
+
+    public List listAfterAdd(){
```

Comment:
This commit doesn't belong to the exceptions exercise.

Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.